packages/cli/api-importers/v2-importer-commons/src/converters/schema/OneOfSchemaConverter.ts

```diff
@@ -126,7 +126,7 @@ export class OneOfSchemaConverter extends AbstractConverter<
                     docs: propertySchema.description,
                     availability: convertedProperty.availability,
                     propertyAccess: await context.getPropertyAccess(propertySchema),
-                    example: undefined
+                    example: convertedProperty.schema?.v2Examples?.userSpecifiedExamples[0]
```

Comment:
do you think the IR should be modeled to be a list here to prevent any lossiness?